### ISSUE TITLE
Custom URL Scheme for tvOS deeplinks

### ISSUE THREAD

**Issue by launsi** [yattee/yattee#553]
### Guidelines

- [X] I have searched the [issue tracker](https://github.com/yattee/yattee/issues) and I haven't found feature request like this

### Type

Improvement to existing feature

### Describe the feature

tvOS currently doesn't support the same custom URL scheme that macOS and iOS support:
yattee://watch?v=dQw4w9WgXcQ

Having this feature in tvOS would be great for automations

**Comment by Deviantgit** [comment]
Agreed, would be super helpful for home assistant.

**Comment by aniqueta** [comment]
Yes, this would be great, as then other apps that enable bookmarking YT videos and such could open Yattee instead of the default YouTube app. Thanks for considering.

**Comment by chris-79** [comment]
+1 🙏
